python/test/model_test.sdf

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
+    <world name="world_test">
+        <model name="model_test">
+            <pose>0.2 0.1 0.5 0.1 0.3 0.4</pose>
+            <static>true</static>
+            <self_collide>true</self_collide>
+            <enable_wind>true</enable_wind>
+
+            <link name="model_link_test_1">
+                <pose>0 0 0 0 0 0</pose>
+            </link>
+
+            <link name="model_link_test_2">
+                <pose>0 0 0 0 0 0</pose>
+            </link>
+
+            <joint name="model_joint_test" type="revolute">
+                <parent>model_link_test_1</parent>
+                <child>model_link_test_2</child>
```

Comment:
I think a `fixed` joint makes more sense in a static model. I think a `revolute` joint also needs an `<axis>`, so this will probably emit an error.

Reply:
Oh I see. After checking a little bit more the [docs](http://sdformat.org/spec?ver=1.10&elem=joint) axis doesn't have any default value as I thought. I'll modify it to a `fixed` joint.

Reply:
Done.